bin/oref0-cron-every-15min.sh

```diff
@@ -12,12 +12,24 @@ assert_cwd_contains_ini
 
 # proper shutdown once the EdisonVoltage very low (< 3050mV; 2950 is dead)
 if is_edison; then
-    sudo ~/src/EdisonVoltage/voltage json batteryVoltage battery | jq .batteryVoltage | awk '{if ($1<=3050)system("sudo shutdown -h now")}' &
+    BATTERY_VOLTAGE="$(sudo ~/src/EdisonVoltage/voltage json batteryVoltage battery | jq .batteryVoltage)"
+    echo "Battery level is $BATTERY_VOLTAGE."
```

Comment:
To differentiate this from the percentage-based alert below, maybe have it echo "Battery voltage is $BATTERY_VOLTAGE." here.

Reply:
Done